src/utils/buildUpdateCache.js

```diff
@@ -0,0 +1,10 @@
+const buildUpdateCache = (mutationName, client) => (mutationResult) => {
```

Comment:
Now that I'm thinking about this updater, what about renaming it to `buildUpdateAuthenticationCache`? It is specifically updating the "authentication" data of the cache and it would be better to be more specific of what this helper does.